src/main/java/org/jboss/aerogear/unifiedpush/message/UnifiedMessage.java

```diff
@@ -248,6 +255,7 @@ public MessageBuilder(Builder builder) {
 
         private final Builder builder;
         private final Message message = new Message();
+        private  WindowsBuilder windowsBuilder;
```

Comment:
since `Message` and `Builder` are final, why is the `WindowsBuilder` different ?